scilpy/reconst/frf.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+import logging
+import numpy as np
```

Comment:
Theoretically, there should be a blank line between system and third party imports.